docs/NF_Dev.md

```diff
@@ -99,17 +103,58 @@ values.
 }
 ```
 
+Additionally, a developer can run the [config python script](../examples/config.py) to deploy multiple network functions, including linear or circular chains of multiple NFs, from a JSON config file. An example config file can be found [here](../examples/example_nf_deploy.json). In the config file, a user can specify a number of seconds for the NFs to run before shutdown with the global "TTL". If no timeout is specified, the NFs will run until a raised error or a manual shutdown (Ctrl + C).
```

Comment:
Put a heading above this like `## Running Groups of NFs` since this isn't really that tightly connected to the config file library (totally different other than using json, in fact)